sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/ContainerRegistryRedirectPolicy.java

```diff
@@ -29,136 +28,131 @@
  */
 public final class ContainerRegistryRedirectPolicy implements HttpPipelinePolicy {
```

Comment:
Long term should this transition to using the azure-core redirect policy?

Reply:
Yes, my guess that we can reuse it directly, but I need to investigate - https://github.com/Azure/azure-sdk-for-java/issues/33183